syntax = "proto3";

package NameServer;